### ISSUE TITLE
Use IIe Keyboards on Apple II Plus 

### ISSUE THREAD

**Issue by dkgeorge** [osiweb/unified_retro_keyboard#45]
Can this project enable the use of Apple IIe Keyboards on the Apple II Plus?

According to the [Apple IIe keyboard pinout documentation](https://github.com/user-attachments/files/15962987/Apple.IIe.keyboard.pinout.txt):
 `4) Unlike the ][ & ][+, the //e keyboard is completely passive with the decoder chip located on the motherboard.`

II've come across references to other projects, such as the [apple-iie-keyboard-ii-encoder-card](https://www.applefritter.com/content/apple-iie-keyboard-ii-encoder-card), which reportedly achieved this compatibility. Unfortunately, detailed information about this project and its author is no longer available.

**Comment by dfnr2** [comment]
Yes, that sort of thing is exactly what this project is intended for.

I'd use the one of the ATMega2560-based encoder boards.  I'd dedicate one row for the SHIFT, CTRL,, and CAPSLOCK keys and enable the weak pullup on the corresponding I/O pins.    That would allow the regular matrix read routine to be used.  The RESET line could get the same treatment, or could be routed directly to the /RESET line.  I prefer the first option, which could allow more control over how a RESET is generated.  If you are willing to eliminate the DIP switches from the encoder, you can also map the OA and CA keys the same way.

Then you'd need to modify one of the keymaps for the Apple 2e matrix.  

All of that is straightforward.  If anyone wants for volunteer to try this, I'm happy to provide guidance.
